### DISCUSSION TITLE
Proposal: do not require <description> in <list> without <term>

### DISCUSSION THREAD

**Discussion by sharwell** [dotnet/csharplang#8976] [Language Ideas]
Champion issue: https://github.com/dotnet/csharplang/issues/8977

## Motivation

Currently, the C# language specification describes the `<item>` documentation element as always requiring the child element `<description>`. This includes cases like numbered and bulleted lists where the `<term>` element is omitted. In these cases, the `<description>` element is unnecessarily verbose and the compiler does not provide an assistance in creating conforming documentation comments.

https://github.com/dotnet/machinelearning/pull/529#discussion_r202801954
https://github.com/dotnet/roslyn/pull/29018

## Detailed design

The C# language specification is updated as follows (final wording TBD):

> If an `<item>` element within a `<list>` element is missing both the `<term>` and `<description>` elements, the inner XML of the `<item>` element is treated as the inner XML of an implicit `<description>` element.

In other words, the following become equivalent:

```xml
<list type="number">
  <item><description>Text of the item</description></item>
</list>

<list type="number">
  <item>Text of the item</item>
</list>
```

## Remarks

This behavior aligns with both user expectations and the long-standing behavior of Sandcastle Help File Builder. This behavior also aligns with the current C# compiler behavior and the IDE experience, so implementation of the proposal will be limited to a change in the wording of the language specification to allow this case.

/cc @EWSoftware @BillWagner @sfilipi
